src/request.spec.ts

```diff
@@ -396,10 +396,14 @@ describe('Request', () => {
   });
 
   it('returns response of a successful GET request, with a proxy url', (done) => {
+    let outcome = false;
     sinon.stub(process, 'env').value({ 'HTTPS_PROXY': 'http://proxy.contoso.com:8080' });
 
     sinon.stub(_request as any, 'req').callsFake((options) => {
       _options = options as CliRequestOptions;
+      if (_options.proxy && _options.proxy.host === 'proxy.contoso.com' && _options.proxy.port === 8080 && _options.proxy.protocol === 'http') {
```

Comment:
We can simplify this code by assigning the outcome of the comparison directly to the `proxyConfigured` variable without an `if`